packages/atomic/src/components/search/result-template-components/atomic-result-timespan/atomic-result-timespan.tsx

```diff
@@ -0,0 +1,98 @@
+import {Schema, StringValue} from '@coveo/bueno';
+import {Result, ResultTemplatesHelpers} from '@coveo/headless';
+import {Component, Prop, State} from '@stencil/core';
+import dayjs from 'dayjs';
+import {DurationUnitType} from 'dayjs/plugin/duration';
+import {InitializeBindings} from '../../../../utils/initialization-utils';
+import {Bindings} from '../../atomic-search-interface/atomic-search-interface';
+import {ResultContext} from '../result-template-decorators';
+import duration from 'dayjs/plugin/duration';
+dayjs.extend(duration);
+
+/**
+ * The `atomic-result-timespan` component renders the duration of a number result field.
+ */
+@Component({
+  tag: 'atomic-result-timespan',
+})
+export class AtomicResultTimespan {
+  @InitializeBindings()
+  public bindings!: Bindings;
+
+  @ResultContext() private result!: Result;
+
+  @State() public error!: Error;
+
+  /**
+   * The result field which the component should use.
+   * This will look for the field in the Result object first, and then in the Result.raw object.
+   * It is important to include the necessary field in the ResultList component.
```

Comment:
I'm not sure what you mean here. Would you have an example of how to include the field in the ResultList component?

Reply:
I used the same phrasing we have for `atomic-result-date`, `atomic-result-text` etc, regarding their own `field` property.

Reply:
Interesting; I see that those descriptions were reviewed over a year ago, and I'm not sure that redoing them would be the highest priority right now (at some point my team should do a full review of the annotations though), but I don't really see what we mean by "include the necessary field". Do you think it means we need to include it in the fields to include?
Not a huge deal, but if we could make an improvement in this PR, it would already be something positive 🙂

Reply:
>  Do you think it means we need to include it in the fields to include?

Yep, that's what it means. We can improve it in this PR, for sure.

Reply:
Oh nice thanks!
Yeah I have a suggestion then; in fact the `fieldsToInclude` prop is deprecated in the ResultList.
```suggestion
   * It is important to include the target field in the `fieldsToInclude` list of the parent `search-interface` component.
```